<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jyotis-MBP" failures="0" tests="5" name="com.applitools.tests.VisualAITests" time="81.980" errors="1" timestamp="2019-11-24T18:00:50 GMT" skipped="0">
  <testcase classname="com.applitools.tests.VisualAITests" name="afterEach" time="0.002">
    <error message="Client instance has been closed." type="java.lang.IllegalStateException">
      <![CDATA[java.lang.IllegalStateException: Client instance has been closed.
at org.glassfish.jersey.internal.guava.Preconditions.checkState(Preconditions.java:169)
at org.glassfish.jersey.client.JerseyClient.checkNotClosed(JerseyClient.java:257)
at org.glassfish.jersey.client.JerseyClient.target(JerseyClient.java:279)
at org.glassfish.jersey.client.JerseyClient.target(JerseyClient.java:56)
at com.applitools.eyes.ServerConnector.closeBatch(ServerConnector.java:722)
at com.applitools.eyes.selenium.SeleniumEyes.closeBatch(SeleniumEyes.java:758)
at com.applitools.eyes.EyesRunner.deleteAllBatches(EyesRunner.java:33)
at com.applitools.eyes.EyesRunner.getAllTestResults(EyesRunner.java:25)
at com.applitools.eyes.EyesRunner.getAllTestResults(EyesRunner.java:17)
at com.applitools.tests.VisualAITests.afterEach(VisualAITests.java:89)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:623)
at org.testng.TestRunner.afterRun(TestRunner.java:853)
at org.testng.TestRunner.run(TestRunner.java:594)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- afterEach -->
  <testcase classname="com.applitools.tests.VisualAITests" name="loginPageUITest" time="20.975"/>
  <testcase classname="com.applitools.tests.VisualAITests" name="loginAndSortTableByAmount" time="22.204"/>
  <testcase classname="com.applitools.tests.VisualAITests" name="testExpenseChart" time="19.579"/>
  <testcase classname="com.applitools.tests.VisualAITests" name="testDynamicContent" time="19.220"/>
</testsuite> <!-- com.applitools.tests.VisualAITests -->
